### PR TITLE
F refactoring sr description multi

### PR DESCRIPTION
**Pull request by mikramarc**
## Proposed changes

Updating xacros and srdf generation to work with new format.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added automated tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [x] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_robot_launch/launch/load_robot_description.launch [hunk 1]
```diff
@@ -2,43 +2,53 @@
   <arg name="sim"/>
 
   <!-- ROBOT CONFIGS -->
-  <arg name="side" default="right"/>
+  <arg name="side"/>
+  <!-- No needed -->
```
Comment (georgiablanco) [inline sr_robot_launch/launch/load_robot_description.launch]: is this comment meant to be here?

File: sr_robot_launch/launch/sr_hardware_control_loop.launch [hunk 2]
```diff
@@ -18,14 +18,18 @@
   <arg name="arm_ctrl" default="$(arg arm)"/>
   <!-- Whether to run hand controllers. -->
   <arg name="hand_ctrl" default="$(arg hand)"/>
-  <!-- Set hand type. -->
+
   <arg name="hand_type" default="hand_e"/>
+  <arg name="hand_version" default="E3M5"/>
+  <arg name="fingers" default="th,ff,mf,rf,lf"/>
```
Comment (georgiablanco) [inline sr_robot_launch/launch/sr_hardware_control_loop.launch:24]: does it matter if this is all or the specified?

File: sr_multi_description/urdf/bimanual_srhand_ur.urdf.xacro [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    <xacro:arg name="left_mid_sensors" default="none"/>
+    <xacro:arg name="left_prox_sensors" default="none"/>
+    <xacro:arg name="left_palm_sensor" default="none"/>
+
+    <xacro:include filename="$(find sr_description)/other/xacro/materials.urdf.xacro"/>
+    <xacro:include filename="$(find sr_description)/hand/xacro/hand.urdf.xacro"/>
+    <xacro:include filename="$(find sr_description)/other/xacro/gazebo.urdf.xacro"/>
+
+    <xacro:sr_hand side="right" hand_type="$(arg right_hand_type)" hand_version="$(arg right_hand_version)"
+                    fingers="$(arg right_fingers)" tip_sensors="$(arg right_tip_sensors)" mid_sensors="$(arg right_mid_sensors)"
```
Comment (beatrizleon) [inline sr_multi_description/urdf/bimanual_srhand_ur.urdf.xacro]:
```suggestion
                   fingers="$(arg right_fingers)" tip_sensors="$(arg right_tip_sensors)" mid_sensors="$(arg right_mid_sensors)"
```

File: sr_multi_description/urdf/bimanual_srhand_ur.urdf.xacro [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                    prox_sensors="$(arg right_prox_sensors)" palm_sensor="$(arg right_palm_sensor)"/>
```
Comment (beatrizleon) [inline sr_multi_description/urdf/bimanual_srhand_ur.urdf.xacro]:
```suggestion
                   prox_sensors="$(arg right_prox_sensors)" palm_sensor="$(arg right_palm_sensor)"/>
```

File: sr_robot_launch/launch/load_robot_description.launch (hunk 1 above)
Comment (beatrizleon) [reply sr_robot_launch/launch/load_robot_description.launch]: we should delete it 
```suggestion
```

File: sr_robot_launch/launch/load_robot_description.launch [hunk 5]
```diff
@@ -2,43 +2,53 @@
...
-  <arg name="hand_type" default="hand_e"/>
   <arg unless="$(arg arm)" name="initial_z" default="0.0"/>
+  <arg name="hand_type"/>
+  <arg name="hand_version"/>
+  <arg name="fingers"/>
+  <arg name="tip_sensors"/>
+  <arg name="mid_sensors"/>
+  <arg name="prox_sensors"/>
+  <arg name="palm_sensor"/>
   <arg name="hand_id" value="$(arg side_letter)h"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/load_robot_description.launch]: Should we rename this hand_prefix?
```suggestion
  <arg name="hand_prefix" value="$(arg side_letter)h"/>
```

Comment (beatrizleon) [reply sr_robot_launch/launch/load_robot_description.launch]: I was thinking that now that we only need this, arm_prefix and side_letter in only one place we can remove the clutter of having this as arg and add them directly in the code.

File: sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch [hunk 6]
```diff
@@ -26,7 +26,22 @@
   <!-- Separation between the robots -->
   <arg name="hand_x_separation" default="-0.4"/>
   <!-- Set the hand type here: hand_e_plus, hand_e, hand_lite, hand_extra_lite -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch]:
```suggestion
  <!-- Set the hand type here: hand_e, hand_g -->
```

File: sr_robot_launch/launch/sr_bimanual_simulation_control_loop.launch [hunk 7]
```diff
@@ -20,7 +20,7 @@
   <arg name="home_angles" default=""/>
 
   <!-- HAND RELATED CONFIG-->
-  <!-- Wheather the hands have biotacs -->
+  <!-- Wheather the hands have biotacs. Needs to be deleted when arm xacros are refactored -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_bimanual_simulation_control_loop.launch]:
```suggestion
```

File: sr_robot_launch/launch/sr_bimanual_simulation_control_loop.launch [hunk 8]
```diff
@@ -20,7 +20,7 @@
...
   <arg name="biotac" default="false"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_bimanual_simulation_control_loop.launch]:
```suggestion
```

**Review by beatrizleon — CHANGES_REQUESTED** [review]

File: sr_robot_launch/launch/sr_hardware_control_loop.launch (hunk 2 above)
Comment (mikramarc) [reply sr_robot_launch/launch/sr_hardware_control_loop.launch:24]: no, they are equivalent but left specificed for the user to understand the format

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**